File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/1020.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');
    
var n = parseInt(lines.shift());

console.log(Math.trunc(n/365)+" ano(s)\n"+Math.trunc((n%365)/30)+" mes(es)\n"+Math.trunc((n%365)%30)+" dia(s)")

File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/1020.java
import java.util.Scanner;

public class Main {
    public static void main(String args[]) {
        Scanner ler = new Scanner(System.in);
        int n = ler.nextInt();
        
        System.out.printf("%d ano(s)\n%d mes(es)\n%d dia(s)\n", n/365, (n%365)/30, (n%365)%30);
    }
}